src/dataDisplay/icon/iconSVG.jsx

```diff
@@ -1,55 +1,107 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import { TITLE_PROP_TYPE } from './iconConstants';
 
 const propTypes = {
-    gradientId: PropTypes.string,
-    maskId: PropTypes.string,
-    pathD: PropTypes.string,
-    pathId: PropTypes.string,
+    circle: PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        cx: PropTypes.string.isRequired,
+        cy: PropTypes.string.isRequired,
+        r: PropTypes.string.isRequired,
+    }),
+    gradientId: PropTypes.string.isRequired,
+    maskId: PropTypes.string.isRequired,
+    path: PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        d: PropTypes.string.isRequired,
+    }),
+    polygon: PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        transform: PropTypes.string,
+        points: PropTypes.string.isRequired,
+    }),
     renderGradientColor: PropTypes.node,
     style: PropTypes.shape({}),
-    title: PropTypes.string,
+    title: TITLE_PROP_TYPE,
     type: PropTypes.string,
     viewBox: PropTypes.string,
 };
 
 const defaultProps = {
-    gradientId: undefined,
-    maskId: undefined,
-    pathD: undefined,
-    pathId: undefined,
-    renderGradientColor: undefined,
-    style: undefined,
-    title: undefined,
-    type: undefined,
+    circle: null,
+    path: null,
+    polygon: null,
+    renderGradientColor: null,
+    style: null,
+    type: null,
+    title: null,
     viewBox: '0 0 24 24',
 };
 
 function IconSVG(props) {
     const {
+        circle,
         gradientId,
         maskId,
-        pathD,
-        pathId,
+        path,
+        polygon,
         renderGradientColor,
         style,
         title,
         type,
         viewBox,
     } = props;
 
+    let id;
+
+    if (circle) {
+        id = circle.id;
+    }
+
+    if (path) {
+        id = path.id;
+    }
+
+    if (polygon) {
+        id = polygon.id;
+    }
+
+    if (type === false) {
+        console.log('## type', type);
+        console.log('id', id);
+    }
```

Comment:
Do we need these log messages?  If so, suppress ESLINT please.  Otherwise please remove them.

Reply:
Cleaned up. Thanks!

Reply:
Yep, looks like they're gone.  Resolving.